tests/NgTests/Data/TestCatalogEntries.resx

```diff
@@ -850,6 +866,10 @@
       "@id": "packageEntry",
       "@container": "@set"
     },
+		"packageTypes": {
```

Comment:
Indentation is inconsistent here

Reply:
Corrected all tabs to spaces. Look consistent for me now. Hopefully will translate correctly here.